vendor/bat-native-ads/src/bat/ads/internal/catalog/catalog_daypart_info.h

```diff
@@ -3,22 +3,22 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BAT_ADS_INTERNAL_CATALOG_CATALOG_DAY_PART_INFO_H_
-#define BAT_ADS_INTERNAL_CATALOG_CATALOG_DAY_PART_INFO_H_
+#ifndef BAT_ADS_INTERNAL_CATALOG_CATALOG_DAYPART_INFO_H_
+#define BAT_ADS_INTERNAL_CATALOG_CATALOG_DAYPART_INFO_H_
 
 #include <string>
 #include <vector>
 
 namespace ads {
 
-struct CatalogDayPartInfo {
-  std::string dow;
+struct CatalogDaypartInfo {
+  std::string days_of_week;
```

Comment:
Should match name used in catalog which I believe is dow

Reply:
I think we should keep it as `days_of_week` as `dow` is at a glance makes it seem to be `day_of_week`

Reply:
We should match catalog/bundle however I agree we should raise tickets to make these fields more verbose on both client/server